### PR TITLE
Update splash page

### PR DESCRIPTION
**Pull request by beetalogist**
hey this is what i did:
altered header to still be fixed position but also extend across page
incorporated arvo font
wrote form for splash page
removed borders from divs

### CODE REVIEW

File: client/js/components/ecosystems/Header.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Header extends React.Component {
   render() {
     return (
       <header className="app-header">
-        <Icon source="/dist/images/logo-white.png" height="36" width="36"/>
+        <Icon source="/dist/images/logo-white-test.svg" height="36" width="36"/>
```
Comment (wbprice) [inline client/js/components/ecosystems/Header.js]: Test?

Comment (wbprice) [reply client/js/components/ecosystems/Header.js]: Please rename the file this refers to and update the reference in this file.

File: client/js/components/ecosystems/PositionLocator.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                    <label htmlFor="GET-name">Name:</label>
+                    <input id="GET-name" type="text"/>
+                    <input type="submit" value="Save"/>
+                </form>
+            </section>
+        )
+    }
+
+}
+module.exports = PositionLocator
```
Comment (wbprice) [inline client/js/components/ecosystems/PositionLocator.js]: This file doesn't get used anywhere and I'm reasonably sure it was just for troubleshooting anyway.  Please remove it.

File: client/js/components/ecosystems/StopFinder.js [hunk 3]
```diff
@@ -2,16 +2,36 @@
...
+                <h1>HRTB.us</h1>
+                <h2>Stop Finder</h2>
+                <div>
+                    <h3>By Stop Number</h3>
                 <form method="GET" action="/api/v1/stopId">
-                    <label for="stopId">Stop Number:</label>
+                    <label htmlFor="stopId">Stop Number:</label>
                     <input id="stopId" name="stopId" type="number" />
-                    <button>Search</button>
+                    <button class="loadMore">Search</button>
```
Comment (wbprice) [inline client/js/components/ecosystems/StopFinder.js]: In JSX, you need to use `className` to add a class to an element.

File: client/js/components/ecosystems/StopFinder.js [hunk 4]
```diff
@@ -2,16 +2,36 @@
 
 const React = require('react');
 const Component = React.Component;
+// const palette = require('./../../palette');
```
Comment (wbprice) [inline client/js/components/ecosystems/StopFinder.js]: In general, you shouldn't leave commented-out code in a PR.  If we're not going to use it for now, please remove this line.

File: client/styles/base.scss [hunk 5]
```diff
@@ -1,12 +1,18 @@
 @import 'palette';
 
-body {
-    font-family: "Roboto", "Helvetica", "sans-serif";
+body, label, button {
```
Comment (wbprice) [inline client/styles/base.scss]: `label` and `button` will inherit their font style from the body, so you shouldn't have to do this.  If there are specific styles you need to apply to labels and buttons, please put those elsewhere.

File: config/web.js [hunk 6]
```diff
@@ -29,6 +29,9 @@ module.exports = {
 
   onPluginsLoaded: function (err) {
     // Note that `this` is Trails `app` instance
+
+    console.log(__dirname);
+
```
Comment (wbprice) [inline config/web.js]: 32-34 can be removed.

**Review by wbprice — CHANGES_REQUESTED** [review]
solid first effort +++

File: client/js/components/ecosystems/Header.js (hunk 1 above)
Comment (beetalogist) [reply client/js/components/ecosystems/Header.js]: updated

File: client/js/components/ecosystems/PositionLocator.js (hunk 2 above)
Comment (beetalogist) [reply client/js/components/ecosystems/PositionLocator.js]: updated

File: client/js/components/ecosystems/StopFinder.js (hunk 3 above)
Comment (beetalogist) [reply client/js/components/ecosystems/StopFinder.js]: updated

File: client/js/components/ecosystems/StopFinder.js (hunk 4 above)
Comment (beetalogist) [reply client/js/components/ecosystems/StopFinder.js]: updated

File: client/styles/base.scss (hunk 5 above)
Comment (beetalogist) [reply client/styles/base.scss]: updated

File: config/web.js (hunk 6 above)
Comment (beetalogist) [reply config/web.js]: updated

**Review by wbprice — APPROVED** [review]
LGTM

**Pull request merged**